src/transactions/ParallelApplyUtils.h

```diff
@@ -102,11 +266,17 @@ class LedgerAccessHelper
     getLedgerEntryOpt(LedgerKey const& key) = 0;
 
     // upsert returns true if the entry was created, false if it was updated.
+    // "created" here is interpreted narrowly to mean there was no live entry in
+    // any parent level of the ledger state; a "local" map-insert that shadows
+    // an existing entry is not considered a create.
     virtual bool upsertLedgerEntry(LedgerKey const& key,
                                    LedgerEntry const& entry) = 0;
 
     // erase returns true if the entry was erased, false if it wasn't present.
-    virtual bool eraseLedgerEntry(LedgerKey const& key) = 0;
+    // as with upsert, this is interpreted narrowly to mean that an insert is
```

Comment:
I don't quite understand this comment, is there a typo? The name suggests that we erase an entry when it exists, not that we insert it.